lib/conekta-button.scss

```diff
@@ -1,242 +1,207 @@
+	overflow-x  : auto;
+	overflow-y  : scroll;
+	padding     : 0 25px;
+	position    : fixed;
+	right       : 0px;
+	top         : 0px;
+	z-index     : 1050;
+
+	.#{$modal_name}-dialog {
+		background    : #FFFFFF;
+		border-radius : 4px;
+		box-shadow    : 0px 5px 15px rgba(0, 0, 0, 0.5);
+		margin        : 104px auto 5%;
+		position      : relative;
+		width         : 413px;
+
+		.#{$modal_name}-content {
+			.#{$modal_name}-header {
+				margin      : 0 26px;
+				padding-top : 22px;
+
+				h1 {
+					font-size      : 27px;
+					letter-spacing : .01em;
+					margin         : 0;
 				}
 			}
 
-			.#{$modal_name}-body{
-				margin: 0 26px;
+			.#{$modal_name}-body {
+				margin : 0 26px;
 			}
 
-			.#{$modal_name}-footer{
-				padding: 20px 26px 13px;
-				border-bottom-right-radius: 4px;
-				border-bottom-left-radius: 4px;
-				
-				background-color: #F9FAFB;
-				border-top: 1px solid #EEEEEE;
-
-
-				button{
-					font-family: $font-family;
-
-					display: inline-block;
-					transition: all 0.15s ease-out 0s;
-					border-radius: 6px;
-					color: white;
-					font-weight: bold;
-					letter-spacing: 1px;
-					margin-bottom: 1em;
-					padding: 1em;
-					text-align: center;
-					border: none;
-					box-shadow: none;
-					text-transform: uppercase;
-
-					&#paynow{
-						background-color: #6D48CC;
-						&:hover{
-							background-color: darken(#6D48CC, 10%);
+			.#{$modal_name}-footer {
+				background-color : #F9FAFB;
+				border-radius    : 0 0 4px 4px;
+				border-top       : 1px solid #EEEEEE;
+				padding          : 20px 26px 13px;
+
+				button {
+					border         : none;
+					border-radius  : 6px;
+					box-shadow     : none;
+					color          : white;
+					display        : inline-block;
+					font-family    : $font-family;
+					font-weight    : bold;
+					letter-spacing : 1px;
+					margin-bottom  : 1em;
+					padding        : 1em;
+					text-align     : center;
+					text-transform : uppercase;
+					transition     : all 0.15s ease-out 0s;
+
+					&#paynow {
+						background-color : #6D48CC;
+						float            : right;
+
+						&:hover {
+							background-color : darken(#6D48CC, 10%);
 						}
-						float: right;
 					}
 
-					&#cancel{
-						background-color: #859093;
-						&:hover{
-							background-color: darken(#859093, 10%);
+					&#cancel {
+						background-color : #859093;
+
+						&:hover {
+							background-color : darken(#859093, 10%);
 						}
 					}
 				}
 			}
-
-
-
-
-
 		}
 	}
 }
 
-.#{$modal_name}-body .card{
-	background: #F9FAFB;
-	border: solid 2px #E2E6E9;
-	padding: 15px;
-	border-radius: 4px;
-	margin-top: 14px;
-	margin-bottom: 35px;
-	.info{
-		position: relative;
-		padding-bottom: 20px;
-
-		.tag{
-			font-size: 14px;
-			color: #FFF;
-			border-radius: 10px;
-			background-color: #838991;
-			display:inline-block;
-			padding-top: 3px;
-			padding-bottom: 3px;
-			padding-left: 6px;
-			padding-right: 6px;
-
+.#{$modal_name}-body .card {
+	background    : #F9FAFB;
```

Comment:
Color `#F9FAFB` should be written as `#f9fafb`<br>Line should be indented with spaces, not tabs<br>Property name should be immediately followed by a colon